source/speechDictHandler/__init__.py

```diff
@@ -118,6 +118,14 @@ def sub(self, text):
 def processText(text):
 	if not globalVars.speechDictionaryProcessing:
 		return text
+	try:
+		# see issue 14689
+		if config.conf["UIA"]["winConsoleImplementation"] == "UIA":
+			stripText = text.rstrip()
+			if len(text)-len(stripText) > 10000:

```

Comment:
Why 10000?

Reply:
I based on @CyrilleB79 's observations in #14689  (comment), and choose it to minimize intervention, but I can surely decrease it a lot.

Reply:
Making further tests, I observed a difference between 17998 and 17834, according to amount of text in console. Now that I use apiLevel on current focused object, however, I think we could reduce it to any value that guarantees no impact on performance and perceived spoken difference.

Reply:
@codeofdusk, do you have suggestions for this value at current stage? Thanks.